tests/MonologCreator/Processor/RequestIdTest.php

```diff
@@ -182,24 +225,31 @@ public function testgenerateBytesWithMtRand()
                 ->disableOriginalConstructor()
                 ->getMock();
 
-        $subject->expects($this->at(0))
-            ->method('isCallable')
-            ->with($this->equalTo('random_bytes'))
-            ->willReturn(false);
-        $subject->expects($this->at(1))
-            ->method('isCallable')
-            ->with($this->equalTo('openssl_random_pseudo_bytes'))
-            ->willReturn(false);
-        $subject->expects($this->at(2))
-            ->method('isCallable')
-            ->with($this->equalTo('mt_rand'))
-            ->willReturn(true);
+        $subject->expects($this->exactly(3))
+        ->method('isCallable')
```

Comment:
From my understanding in chained calls any following line that continues the call should be indented by one level

Reply:
fixed